docs/docs/deployment/hosting/aptible.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Aptible
+---
+
+## Prerequisites
+
+The options and health check routes described in this document are available from Flagsmith 2.130.0.
```

Comment:
```suggestion
The options and health check routes described in this document are available from Flagsmith `2.130.0`.
```

Reply:
There's no right or wrong way to do this, but I prefer the advice from Google's technical documentation style guide - version numbers are typically not styled in code font. See https://developers.google.com/style/code-in-text